datafusion/core/tests/sqllogictests/test_files/timestamps.slt

```diff
@@ -837,6 +837,16 @@ SELECT DATE_TRUNC('YEAR', TIMESTAMP '2022-08-03 14:38:50Z');
 ----
 2022-01-01T00:00:00
 
+query P
+SELECT DATE_TRUNC('year', NULL);
+----
+1970-01-01T00:00:00
```

Comment:
Hmm, I think @alamb added the expected behavior in #6701, which is `null should be returned` instead of this default value.